python/torch_mlir_e2e_test/test_suite/reduction.py

```diff
@@ -572,6 +572,77 @@ def ReduceAmaxKeepDim_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class ReduceMinAllDims(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, a):
+        return torch.ops.aten.min(a)
+
+@register_test_case(module_factory=lambda: ReduceMinAllDims())
+def ReduceMinAllDims_basic(module, tu: TestUtils):
+    module.forward(tu.rand(3, 4, 5, low=-10, high=-5))
+
+# ==============================================================================
+
+class ReduceMinFloatModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, a):
+        return torch.ops.aten.min(a)
+@register_test_case(module_factory=lambda: ReduceMinFloatModule())
+def ReduceMinFloatModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(3, 4, 5))
+
```

Comment:
I don't see any difference in both these tests, except the former one has limits for the random values. I think we need only one of these.

Reply:
You're correct, I copied this set of tests from the corresponding tests for `AtenMaxOp`, but I'll remove one.